pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "temporian"
-version = "0.1.0"
+version = "0.1.3" # TODO: roll back to 0.1.0
```

Comment:
Why not just move to 0.1.1 and release a new version with updated authors, maintainers etc?

Reply:
I figured we could do that when we decided to do a new release - change the version name in main, then merge to release